src/core/server/tools/identity.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+import { loadConfig } from '../../storage/config.js';
+import { Signer } from '@ucanto/principal/ed25519';
+
+export const identityTool = {
+  name: 'identity',
+  description: 'Returns the DID key of the Storacha agent loaded from the private key storage config.',
+  inputSchema: z.object({}),
+  handler: async () => {
+    try {
+      const config = loadConfig();
```

Comment:
Is there no way to load this once? If no then perhaps memorize `loadConfig` so subsequent calls don't have to do all the parsing and validation.

Reply:
yes, we can load it when we start the MCP server. Just changed that. thanks